Code-4-Community/speak-for-the-trees-frontend
# Create user home

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/pulses/816228808)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

The UI for users to navigate the site easily, currently just the styling.

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

Updated Home.tsx and created home.less to match the designs
- Used styled components to override internal styles
- Used pageheader component for the page header and subheader
- Created and used linkcard component
  - Used linkbutton component and styled components
  - All links lead to the landing page for now

## Screenshots

Provide screenshots of any new components, styling changes, or pages.
<img width="1680" alt="UserHome" src="https://user-images.githubusercontent.com/22990100/99922486-d57d3500-2cfe-11eb-8dec-3bce5e8c789d.png">

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.
- Visually checked that the page matches the designs
- Shrunk/maximized window size to check flexibility
- Clicked links to check functionality
